### DISCUSSION TITLE
Nivo line graph acting strange

### DISCUSSION THREAD

**Discussion by imtiazhussain110** [plouc/nivo#2263] [Q&A] [answered]
<img width="928" alt="nivo-line" src="https://user-images.githubusercontent.com/96948430/224530191-a9f52db6-543d-4797-a545-df0497418a6c.PNG">

Hey there! I'm creating some line graphs with the help of nivo charts. They were working good until now. I didn't change the code but today when I ran the code I saw this. A blue color showed along with graph lines, I don't know if it is feature or what. I tried even copying the code from nivo chart's wesite and pasted it as it is in the .jsx file but I got the same result. I want to remove that. Please help me understand it why it is happening and how can I remove it. Thank you!

**Comment by tkonopka** [comment, marked as answer]
If these are svg charts, it may be that svg path components are using a solid color as a fill. You can try to set prop `fill='none'` on your Line component. But if this appeared without any changes to the nivo code, it may be caused by a global style rule elsewhere on your page/site that is now affecting these two graphs. In that case, track down the rule and edit it to avoid hitting the nivo charts. Alternatively, add a new style rule, e.g. `path { fill: 'none', fill-opacity: 0 }`, so that it affects these two charts and does not upset other elements on the page/site.

**Reply by imtiazhussain110** [reply]
thank you @tkonopka. It helped. It was another style rule I was applying on another graph and then forgot about it. Now I know what caused it.
